src/surabhi/assignment12/StudentMarks.java

```diff
@@ -0,0 +1,61 @@
+package surabhi.assignment12;
+
+public class StudentMarks {
+
+		double sum(double num1, double num2, double num3) {
+			return num1+num2+num3;
+		}
+		
+		double average(double num1, double num2, double num3) {
+			return (num1+num2+num3)/3;
+		}
+		
+		boolean isAnswerAboveExpecation(double avg) {
+			if(avg>=50) {
+				return true;
+			}else {
+				return false;
+			}
+
+		}
+		
+		static boolean isEligible(double avg) {
+			if(avg>=50 || avg%2==0) {
```

Comment:
we need to use divisible function not modular.
if value is more than 50 or divisible by 2. Eligibility will be printed in main method.